lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2809,24 +2809,20 @@ ParseResult FIRStmtParser::parseInstance() {
   }
 
   InstanceOp result;
-  if (getConstants().options.rawAnnotations) {
```

Comment:
Shouldn't the parser option go away?

Reply:
Yes, this should be totally gone now. 👍

Reply:
I removed `rawAnnotations` from the parser options in the force-push.